src/waveform/waveformwidgetfactory.cpp

```diff
@@ -1168,14 +1171,17 @@ QString WaveformWidgetFactory::buildWidgetDisplayName() const {
         extras.push_back(tr("legacy"));
     }
     if (isOpenGlAvailable() || isOpenGlesAvailable()) {
-        if (WaveformT::useOpenGLShaders()) {
+        if (WaveformT::useOpenGles()) {
+            if (WaveformT::useOpenGLShaders()) {
+                extras.push_back(QStringLiteral("GLSL ES"));
+            } else {
+                extras.push_back(QStringLiteral("GLES"));
```

Comment:
we now have a bunch of shaders being compatible with plain openGL and OpenGL ES, what "backend" does it choose by default?

Reply:
Right, that strategy makes sense, but it does not reflect how the name is chosen here where ES is prioritized just based on what the shader supports.

Reply:
The name is dynamically changed according to the context that is used.  You will see 
"RGB (GLSL ES)" on a Raberry Pie and "RGB (GLES)" on your Laptop.

Reply:
Oh I see that is currently broken. I will fix it.